lib/ui/painting.dart

```diff
@@ -2079,6 +2079,16 @@ class Path extends NativeFieldWrapperClass2 {
   }
   Path _transform(Float64List matrix4) native 'Path_transform';
 
+  /// Cuts the path into a subset of itself leaving only the region between
+  /// `startT` and `stopT` fraction values. Set `isComplement` to true to
+  /// invert the result by cutting out the region from `startT` to `stopT`.
+  ///
+  /// Returns true when the cut is succesfully made, false if not.
+  bool trim(double startT, double stopT, bool isComplement) {
```

Comment:
Please document the valid ranges and also clamp before making the native call.

Reply:
Sounds good, I changed it to the same range provided in the Skia docs. I originally didn't clamp them as Skia does this internally, but I did make the change now!